src/Components/test/E2ETest/Tests/ComponentRenderingTest.cs

```diff
@@ -400,6 +400,26 @@ public void CanUseJsInteropToReferenceElements()
             Browser.Equal("Clicks: 2", () => inputElement.GetAttribute("value"));
         }
 
+        [Fact]
+        public void CanUseFocusExtensionToFocusElement()
```

Comment:
Could you add a test that calls `FocusAsync` on an `ElementReference` that is read from a JSInteop call? That should verify that the service provider is being correctly hydrated by the converter.

Reply:
We realized this isn't a real scenario.